guides/common/modules/proc_acd_installation.adoc

```diff
@@ -0,0 +1,50 @@
+[id="{context}_installation"]
+= Installation
+
+You need to install both the `foreman_acd` and `smart_proxy_acd` plugins to use ACD.
+
+This describes the package installation on CentOS 7, Red Hat Enterprise Linux 7, and Oracle Linux 7.
+ifdef::foreman-deb[]
+Running ACD on Debian and Ubuntu is currently untested.
+endif::[]
+
+ifdef::foreman-el,katello[]
+This guide assumes you already have a working https://docs.theforeman.org/nightly/Installing_Server_on_Red_Hat/index-foreman-el.html[Foreman] or https://docs.theforeman.org/nightly/Installing_Server_on_Red_Hat/index-katello.html[Foreman and Katello] installation.
+endif::[]
+
+.Procedure
+ifdef::foreman-el,foreman-deb,katello,satellite[]
+. Create a new yum repository `/etc/yum.repos.d/foreman-plugins.repo` on your {ProjectServer}:
++
+[source,bash,subs="+quotes,attributes"]
+----
+[foreman-plugins]
+name=Foreman plugins
+baseurl=https://yum.theforeman.org/plugins/nightly/el7/x86_64/
+enabled=1
+gpgcheck=0
+----
+endif::[]
+. Install the required packages on your {ProjectServer}:
++
+[source,bash]
+----
+yum install tfm-rubygem-foreman_acd tfm-rubygem-smart_proxy_acd tfm-rubygem-smart_proxy_acd_core
+----
+. Run database migrations on your {ProjectServer}:
++
+[source,bash]
+----
+foreman-rake db:migrate
+foreman-rake db:seed
+----
+. Restart the {ProjectName} services:
++
+[source,bash,subs="+quotes,attributes"]
+----
+{foreman-maintain} service restart
+----
+
+ifdef::foreman-el,foreman-deb,katello,satellite[]
+Refer to the https://theforeman.org/plugins/#2.Installation[Foreman plugin installation] guide for more information.
```

Comment:
We explicitly use `Refer to X for more information about/on how to do Y` in orcharhino documentation.

I personally dislike `For more information` because it doesn't really feel like a complete sentence.

Nevertheless, fixed (with a dot at the end).